openedx/features/caliper_tracking/tests/expected/showanswer.json

```diff
@@ -0,0 +1,44 @@
+{
+    "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+    "action": "Viewed",
```

Comment:
`Showed` is a more suitable action.

Reply:
In that case we will have to set the `event_type` to `event` because `Showed` is not supported in any event type and also I think `ViewEvent` is more suitable.

Reply:
Alright.